### PR TITLE
Github pages for graphs

### PR DESCRIPTION
**Pull request by az85252**
How should I say that we need dataprofiler[reports] to be installed for requirements?

### CODE REVIEW

File: docs/source/graphs.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
+.. _reports:
+
+Graphs
+********
+
+Graph Your Data
+=================
+
+We can plot some of our data as seaborn histogram plots. Below will demonstrate how to do so and provide examples.
```
Comment (JGSweets) [inline docs/source/graphs.rst:9]: maybe we should list the plots we have available (just 1 right now)
```
The following plots are currently available to work directly with your profilers:

 * histogram (numeric columns only)

```

Below we will demonstrate how to do so with examples.

File: docs/source/graphs.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            [2, 'b', 2.2],
+            [3, 'c', 3.5],
+            [None, 'd', 10.0]]
+    profiler = dp.StructuredProfiler(data)
+
+    # This will plot all IntColumn and FloatColumn as histograms (The first and last column).
+    graphs.plot_histograms(profiler)
+
+    # This will plot the specified, column 0, as a histogram.
+    column = [0]
```
Comment (JGSweets) [inline docs/source/graphs.rst:67]: says column but below uses columns

File: docs/source/graphs.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+These are what the variables mean:
+
+    * **column** - The IntColumn or FloatColumn we want to plot
+    * **axes** - (Optional) The axes we want to specify.
+    * **title** - (Optional) The title of the plot we want to define.
+
+Examples
+~~~~~~~~~~~~~~~~~
+
+1. This example demonstrates how we can take a StructuredProfiler class and plot histograms.
```
Comment (JGSweets) [inline docs/source/graphs.rst:51]: `... and plot histograms of the specified columns`

File: docs/source/graphs.rst [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    profiler = dp.StructuredProfiler(data)
+
+    # This will plot all IntColumn and FloatColumn as histograms (The first and last column).
+    graphs.plot_histograms(profiler)
+
+    # This will plot the specified, column 0, as a histogram.
+    column = [0]
+    graphs.plot_histograms(profiler, columns)
+
+* If a name is specified for a column, and you want to put that column as to be plotted, you would put the name of the column in columns. In this case, 0 is the name of the first column.
```
Comment (JGSweets) [inline docs/source/graphs.rst:70]: prob can delete line as you describe in the code comments

File: docs/source/graphs.rst [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+2. This example demonstrates how we can plot a specific histogram.
```
Comment (JGSweets) [inline docs/source/graphs.rst:72]: how we can plot a the histogram of a low level profiler.

File: docs/source/graphs.rst [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+.. code-block:: python
+
+
+    data = pd.Series([1, 2, 3], dtype=str)
+    profiler = IntColumn('example')
+    profiler.update(data)
+
+    # We will plot a IntColumn as a histogram
+    graphs.plot_col_histogram(self.profiler)
```
Comment (JGSweets) [inline docs/source/graphs.rst:82]: just `profiler`

**Review by AnhTruong — COMMENTED** [review]
lgtm

File: docs/source/graphs.rst [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+    * **title** - (Optional) The title of the plot we want to define.
+
+Examples
+~~~~~~~~~~~~~~~~~
+
+1. This example demonstrates how we can take a StructuredProfiler class and plot histograms of the specified columns.
+
+.. code-block:: python
+
+    import dataprofiler as dp
```
Comment (JGSweets) [inline docs/source/graphs.rst:61]: for full example, also need to import graphs in this example,
`from dataprofiler.reports import graphs`

File: docs/source/graphs.rst [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+    columns = [0]
+    graphs.plot_histograms(profiler, columns)
+
+
+2. This example demonstrates how we can plot a low level profiler..
+
+.. code-block:: python
+
+
+    data = pd.Series([1, 2, 3], dtype=str)
```
Comment (JGSweets) [inline docs/source/graphs.rst:82]: specify the imports here too so user can just copy paste

**Review by JGSweets — APPROVED** [review]

**Comment by JGSweets** [comment]
FYI, the images will have to be uploaded in a separate PR.

**Pull request merged**